docs/Demo_Projects/Edge_Impulse_demo.md

```diff
@@ -0,0 +1,204 @@
+
+![Snake motion detector](images/thingy91-snake.gif)
+
+*Detecting the motion patterns: "idle" and "snake"*
+
+LwM2M Objects used:
+
+- Security: `/0` 
+- Server: `/1`
+- Device: `/3`
+- Pattern Detector: `/33650` (custom object)
+
+## Prerequisites
+- Thingy:91
+- [Coiote IoT DM](https://eu.iot.avsystem.cloud/) account
+- Installed <a href="https://docs.zephyrproject.org/latest/getting_started/index.html" target="_blank">Zephyr</a> dependencies
+- Serial communication program e.g. minicom or RealTerm (for Linux or Mac) or PuTTy (for Windows)
+
+!!! warning
+    This tutorial uses the Arm Embedded Toolchain, which is not compatible with the Apple M1 chipset.
+
+## Compilation
+
+### Clone the Anjay Zephyr repository
+
+Open the command line interface on your machine and clone the [Anjay Zephyr repository](https://github.com/AVSystem/Anjay-zephyr-client) using [Git](https://formulae.brew.sh/formula/git):
+
+```jsx
+git clone https://github.com/AVSystem/Anjay-zephyr-client
+```
+
+Set West manifest path to `Anjay-zephyr-client/ei_demo`, and manifest file to `west-nrf.yml` and run `west update`:
+
+```jsx
+west config manifest.path Anjay-zephyr-client/ei_demo
+west config manifest.file west-nrf.yml
+west update
+```
+
+### Update the endpoint name and PSK (optional)
+
+!!! info
+    **The default credentials are**:
+
+    * Endpoint name: `Anjay-zephyr-client`
+    * PSK (Pre-Shared Key): `test`
+
+The Endpoint Name and PSK can be configured using *menuconfig*.
+
+To open *menuconfig*, run the following command:
+```bash
+west build -b thingy91_nrf9160_ns -p -t menuconfig
+```
+
+Open up the folder `anjay-zephyr-client --->`
+
+![menuconfig1.png](images/menuconfig_ei1.png)
+
+Create a new **Endpoint Name** and **PSK Key**:
+
+![menuconfig2.png](images/menuconfig_ei2.png)
+
+After making the changes, press `Q` to close the config menu and `Y` to save.
+
+### Compile the project
+
+Go to the `ei_demo` directory and compile the project for the Thingy:91:
+
+```jsx
+west build -b thingy91_nrf9160_ns
+```
+
+## Write the firmware to the Thingy:91
+
+1. Find the build file in the directory `/build/zephyr`, named: `app_signed.hex`.
+1. Connect the Thingy:91 to a USB port of your machine.
+1. Flash the file using the **Programmer** application in **nRF Cloud for Desktop** via **MCUboot**.
+
+*Find more information on flashing the Thingy:91 using MCU Boot <a href="https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#program-the-nrf9160-sip-application" target="_blank">here</a>.*
+
+
+## Connect the Thingy:91 to Coiote IoT Device Management
+
+To connect to Coiote IoT Device Management LwM2M Server, please register at [eu.iot.avsystem.cloud](https://eu.iot.avsystem.cloud/).
+
+To connect the board:
+
+1. [Log in](https://eu.iot.avsystem.cloud/) to Coiote IoT DM and select **Device Inventory** from the left side menu.
```

Comment:
Delete the hyperlink here. We don't need this

Reply:
IMO we can keep the link. It doesn't hurt to keep it, and it may only help people to find the link to Coiote sooner.

Reply:
Then I can approve the whole pull request